src/nvim/api/ui.c

```diff
@@ -314,15 +314,15 @@ void nvim_ui_try_resize_grid(uint64_t channel_id, Integer grid, Integer width,
   ui_grid_resize((handle_T)grid, (int)width, (int)height, err);
 }
 
-/// Tell Nvim the number of element displayed in popumenu. The amount of
-/// movement by <PageUp> or <PageDown> is determined by the value set by this.
+/// Tell Nvim the number of elements displayed in the popumenu.
+/// The amount of movement by <PageUp> or <PageDown> is determined by the value
+/// set by this.
 ///
-/// If the ext_popupmenu option is false or the height is 0 or less, fails
-/// with error.
+/// Using a height of 0 or less causes an error.
 ///
 /// @param channel_id
 /// @param height  The popupmenu height.
-/// @param[out] err Error details, if any
+/// @param[out] err Error details, if any.
```

Comment:
This kind of change should be done in "all or none". I.e., all of the `@param[out] err` docstrings should be the same.